styles.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+@import url('https://elegantui.netlify.app/styles.css');
+
+html{
+    font-size: 100%;
```

Comment:
font size by default 100% you don't have to define it here

Reply:
changes done